db/models.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy import Column, Integer, String, ForeignKey, Date
+from sqlalchemy.orm import relationship
+
+from db.database import Base
+
+
+class DBBook(Base):
+    __tablename__ = "book"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String(255), nullable=False)
+    summary = Column(String(511), nullable=False)
+    publication_date = Column(Date, nullable=False)
+    author_id = Column(Integer, ForeignKey("author.id"))
+
+
+class DBAuthor(Base):
+    __tablename__ = "cheese"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255), nullable=False, unique=True)
+    bio = Column(String(255), nullable=False)
+    book = relationship(DBBook)
```

Comment:
The `book` relationship should be renamed to `books` to better reflect the one-to-many relationship between authors and books.